packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -106,8 +106,7 @@ describe('Select', () => {
             return fixture<Select>(viewTemplate);
         }
 
-        // Disabled due to intermittancy, see: https://github.com/ni/nimble/issues/1172
-        xit('should limit dropdown height to viewport', async () => {
+        it('should limit dropdown height to viewport', async () => {
             const { element, connect, disconnect } = await setup500Options();
             await connect();
             const listbox: HTMLElement = element.shadowRoot!.querySelector('.listbox')!;
```

Comment:
I think from below we can remove the long comment and workaround of reaching into the parent iframe and calling scrollTo

Reply:
Actually, that's still relevant/necessary. The repositioning we're doing now is still within the context of the iframe. We still have to scroll so that the full iframe is visible.

Reply:
@m-akinc 
Isn't the script wrong then?
it scrolls to the bottom of the iframe but now our stuff is at the top `window.parent.scrollTo(0, window.parent.document.body.scrollHeight);`

Reply:
We're scrolling the Karma header out of view so that the top left corner of the iframe is at the top left corner of the viewport.
![image](https://user-images.githubusercontent.com/7282195/234649596-47900491-61e1-456b-91df-a15f25b9ce5c.png)
The iframe is the same size as the viewport (which is why we have to scroll to get it all visible).

Reply:
https://github.com/ni/nimble/issues/1208